GOVERNANCE.md

```diff
@@ -1,172 +1,73 @@
-<!-- see https://github.com/yzhang-gh/vscode-markdown/blob/master/README.md#table-of-contents -->
-<!-- omit in toc -->
-# GitOps Working Group Governance
+# Governance
 
-This document <https://github.com/gitops-working-group/gitops-working-group/blob/main/GOVERNANCE.md> defines the governance process for the OpenGitOps project and the GitOps Working Group and community.
+This document defines the governance process for the GitOps Working Group and community.
 
-- [Values](#values)
-  - [Code of Conduct](#code-of-conduct)
-- [Roles in the GitOps Working Group Community](#roles-in-the-gitops-working-group-community)
-  - [Users](#users)
-  - [Contributors](#contributors)
-  - [Maintainers](#maintainers)
-  - [Oversight Committee](#oversight-committee)
-- [Decision Making](#decision-making)
-  - [Deciders](#deciders)
-  - [Decision Guidelines](#decision-guidelines)
-  - [Simple Majority Decisions](#simple-majority-decisions)
-  - [Supermajority Decisions](#supermajority-decisions)
-  - [Unanimity Decisions](#unanimity-decisions)
-- [Proposal Process](#proposal-process)
-- [Licenses and Copyright](#licenses-and-copyright)
+## Membership
 
-## Values
+Membership is self-declared.
```

Comment:
"While anyone may join and contribute, voting members must be active members of the CNCF. "